conf.default.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Yeah, I just saw you stuff in src/config.js (sry had some mtgs mid-review)

please update the README, I'm on the fence on storing config as json. Is there a reason you didn't want to use something like dotenv? https://www.npmjs.com/package/dotenv